data/human/syndicate missions.txt

```diff
@@ -0,0 +1,136 @@
+mission "Weapon Salesman"
```

Comment:
you forgot the copyright text we have in at the start of all files here, and everywhere else